community/code-of-conduct.md

```diff
@@ -0,0 +1,97 @@
+# Music Encoding Initiative Community Code of Conduct
+
+## **1. Purpose**
```

Comment:
here and at all other headings the asterisks for marking text bold are not needed, as the styles of the page will automatically render them bold.

```suggestion
## 1. Purpose
```